00-locals.tf

```
@@ -59,17 +64,18 @@ locals {
         registerWithFQDN = true
       },
       certSANs = [
-        module.elb_k8s_elb.elb_dns_name
+        module.elb_k8s_elb.elb_dns_name,
       ],
       kubelet = {
         extraArgs = {
           rotate-server-certificates = true
+          cloud-provider             = "external"
         }
       }
     }
   }
 
-  # Used to configure Cilium Kube-Proxy replacement
+  # Used to configure Cilium Kube-Proxy replacement  
```

```suggestion
  # Used to configure Cilium Kube-Proxy replacement
```

Remove trailing whitespaces